PA_Tests.cls

```diff
@@ -0,0 +1,194 @@
+        ApexPages.StandardController sc = new ApexPages.standardController(PA);
+        PartnerAgreement_ext ext =  new PartnerAgreement_ext(sc);
+		
+		User testUser = [select id,Name, Email, UserName from User where isActive =true and lastname = 'testing' and firstname='tester' limit 1];
+
+		ext.getvSpecialists();
+        //ext.MyAgreement.Agreement_Type__c = 'Mugs & Bowls';
+		ext.MyAgreement.Agreement_Type__c = 'TestData';
+        ext.RefreshProds(); //calls FieldVisibilityControl()
+
+		//PrdColors test input
+		list<SelectOption> options = ext.getvSpecialists();
+        test.starttest();
+        	//ext.getRepGroups();
+          map<string,Partner_Agreement_Healthcare_Specialist__c> mSpecialists = Partner_Agreement_Healthcare_Specialist__c.getAll();		    
+
+          system.assertnotequals(null,options,'Options should be instantiated');
+          system.assert(!options.isEmpty(),'Options should be populated');
+          string vlabel;
+          for(integer i=0; i<options.size(); i++){
+          vlabel = options.get(i).getvalue();
+          system.assertnotequals(null, vlabel,'Label shouldnt be empty');
+          if(vlabel == 'gwilson@WizardCast.com.test'
+            || vlabel =='pwilson@WizardCast.com.test'
+            || vlabel =='rwilson@WizardCast.com.test'|| vlabel == testUser.email 
+            ||  vlabel =='--none--' || vlabel == ''){
+
+            }else{
+              system.assert(false,vlabel +' HealthSpecialst does not match custom setting values');
+            }
+          }
+        ext.vSpecialist = testUser.Email;
+        string vUserID = ext.queryHealthcareSpecialist();
+        system.assertequals(vUserID,testUser.id);
+        test.stoptest();   
+
+	}
+	@isTest static void test_CMDT_Coverage(){
+
+		Partner_Agreement__c PA = new Partner_Agreement__c();
+        PageReference pageRef = Page.PartnerAgreements;
+        Test.setCurrentPage(pageRef);
+        ApexPages.StandardController sc = new ApexPages.standardController(PA);
+        PartnerAgreement_ext ext =  new PartnerAgreement_ext(sc);
+		
+		User testUser = [select id,Name, Email, UserName from User where isActive =true and lastname = 'testing' and firstname='tester' limit 1];
+
+		ext.getvSpecialists();
+        //ext.MyAgreement.Agreement_Type__c = 'Mugs & Bowls';
+		ext.MyAgreement.Agreement_Type__c = 'TestData';
+    ext.RefreshProds(); 
+    ext.vSpecialist = testUser.UserName;   //set the HealthSpecialist      
+    ext.MyAgreement.Agreement_Type__c = 'Mugs';
+    ext.FieldVisibilityControl();       
+    system.assertequals(true,ext.DisplayDiameter );
+    ext.MyAgreement.Agreement_Type__c = 'Tumblers';
+    ext.FieldVisibilityControl();
+    system.assertequals(true,ext.DisplayDisposableLidCode );
+    ext.MyAgreement.Agreement_Type__c = 'Camlids';
+    ext.FieldVisibilityControl();
+    system.assertequals(true,ext.DisplayShoreline );
+
+    ext.getRepGroups();
+		//PrdColors test input
+    test.starttest();
+
+
```

Comment:
Perhaps this is the result of a bad branch merge? 
When we write tests, we want to make sure that we're asserting the resulting values to ensure the code is working as we expect. 
Before we can merge this, we need to address the lack of assertions.